normandy/recipes/tests/test_filters.py

```diff
@@ -206,6 +208,54 @@ def test_throws_error_on_bad_platform(self):
             filter.to_jexl()
 
 
+class TestAddonInstalledFilter(FilterTestsBase):
+    def create_basic_filter(self, addons=["@abcdef", "ghijk@lmnop"], any_or_all="any"):
+        return AddonInstalledFilter.create(addons=addons, any_or_all=any_or_all)
+
+    def test_generates_jexl_installed_not_active(self):
```

Comment:
I the the names of these tests are out of date? It looks like they may have been written before the filter was split into two.